_posts/01-02-01-Use-the-Current-Stable-Version.md

```diff
@@ -9,7 +9,7 @@ anchor:  use_the_current_stable_version
 If you are getting started with PHP, start with the current stable release of [PHP 7.1][php-release]. PHP 7.1 is very
 new, and adds many amazing [new features](#language_highlights) over the older 5.x versions. The engine has been largely re-written, and PHP is now even quicker than older versions.
 
-Most commonly in the near future you will find PHP 5.x being used, and the latest 5.x version is 5.6. This is not a bad option, but you should try to upgrade to the latest stable quickly - PHP 5.6 [will not receive security updates beyond 2018](http://php.net/supported-versions.php).  Upgrading is really quite easy, as there are not many [backwards compatibility breaks][php71-bc]. If you are not sure which version a function or feature is in, you can check the PHP documentation on the [php.net][php-docs] website.
+Most commonly in the near future you will find PHP 5.x being used, and the latest 5.x version is 5.6. This is not a bad option, but you should try to upgrade to the latest stable quickly - PHP 5.6 [will not receive security updates beyond 2018](http://php.net/supported-versions.php).  Upgrading is really quite easy, as there are not many [backwards compatibility breaks][php71-bc]. If you are not sure which version a function or feature is in, you can check the PHP documentation on the [php.net][php-docs] website. Migrating to PHP 7.0 is discouraged because it's active development will cease in December 2017.
```

Comment:
Tiny tweak:

The `it's` in `because it's active development will cease` should be `its` (no apostrophe).

(`It's` always means `it is` or `it has`, which would of course read strangely in this sentence.)

Reply:
We should redo this whole thing, "Most commonly in the near future you will find PHP 5.x being used" does not seem like its a valid statement anymore? I wrote that two years ago.

Reply:
Plus, the latest stable is 7.1, as mentioned in the previous paragraph. Given that the PHP 5 info should probably go, there doesn't remain anything in this paragraph that isn't already stated. Perhaps this paragraph should be deleted instead?

Reply:
@drbyte doh! It hurts my eyes now that I see it, sorry! :)

@philsturgeon wouldn't be so sure. I've been job hunting lately and apart from industry leaders I've seen many job ads stating "we're using 5.6, but planning to go to 7". I've also seen "..planning to go to 7.0", which pushed me to add the note in the first place :) It felt that the paragraph captured the current state of things quite good (which is actually sad). 

I love PRs that just remove dead code, so I don't mind removing it completely instead. It's just a question of what's the guide supposed to do - state the status of things or push best practices? It's supposed to be the list of best practices, so saying "just use latest stable" is probably correct.

What about replacing the whole paragraph with a link to the migration guide as people may not be aware that such a thing exists. Something like "Migration paths are [described in the PHP manual](http://php.net/manual/en/appendices.php)"?

Reply:
Changes added in new commits. If it's acceptable like this I'll squash it to keep the history nice and clean...